pages/cart.js

```diff
@@ -0,0 +1,56 @@
+const { I } = inject();
+
+module.exports = {
+  totalPriceWithoutTax: { css: '#total_price_without_tax' },
+  proceedToCheckoutSummaryButton: "//p/a[@title='Proceed to checkout']",
```

Comment:
use `{ xpath: "//p/a[@title='Proceed to checkout']" }` to indicate `xpath` locator